app/build.gradle

```diff
@@ -33,33 +33,38 @@ android {
     }
 
     buildTypes {
-        release {
+        debug {
+            shrinkResources false
+            minifyEnabled false
+            applicationIdSuffix ".debug"
+            manifestPlaceholders.isRaptorEnabled = "true"
+        }
+        release { // This isn't deployed to Google Play, but is used as a template for "real" releases like "beta"
             shrinkResources true
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            matchingFallbacks = ['release']
         }
-        releaseRaptor {
+        releaseRaptor { // "releaseRaptor" is only used for performance testing, and isn't a real "release" type
             initWith release
             manifestPlaceholders.isRaptorEnabled = "true"
-            matchingFallbacks = ['release']
         }
-        debug {
-            shrinkResources false
-            minifyEnabled false
-            applicationIdSuffix ".debug"
-            manifestPlaceholders.isRaptorEnabled = "true"
+        greenfieldNightly {
+            initWith release
+        }
+        greenfieldBeta {
+            initWith release
+            applicationIdSuffix ".beta"
```

Comment:
If we drop the "firefox" variants then let's drop the "greenfield" part here too.

Reply:
I'm 100% on board, but be advised that [yesterday we talked about keeping the `greenfield` moniker around for a bit](https://mozilla.slack.com/archives/CA4QR1M2R/p1555427349130900)

Reply:
Yeah, but this was in conjunction with the `firefox` types. If we don't keep them then there's no reason to keep `greenfield`. :)